packages/headless/src/controllers/facets/automatic-facet/headless-automatic-facet.ts

```diff
@@ -40,14 +40,14 @@ export interface AutomaticFacetProps {
 }
 /**
  * The `AutomaticFacet` controller allows you to create a search interface component that the end user
- * can use to refine a query by selecting filters based on item metadata (i.e., field values). Unlike traditional facets that
+ * can use to refine a query by selecting filters based on item metadata (i.e., field values). Unlike regular facets that
  * need to be explicitly defined and requested in the query, automatic facets are dynamically generated by the index
  * in response to the query.
```

Comment:
The `AutomaticFacet` controller is used internally by the `AutomaticFacetGenerator` controller to manage the individual state of each dynamically generated facet.

To learn more about the automatic facet generator feature, see [About the Facet Generator](https://docs.coveo.com/en/n9sd0159/).

**Important:** This controller is meant for internal use by Headless only. As an implementer, you should never import or use this controller directly in your code.